### DISCUSSION TITLE
Parameters value

### DISCUSSION THREAD

**Discussion by gigli69** [BobanSpasic/PSS-Revive#12] [Ideas]
In the VMEM view, is it possible, on the right side, to display the value of the parameter, after modifying on the left side with the mouse?

**Comment by BobanSpasic** [comment]
It is possible, but it would not look nice. The Grid component used on the right side will always align the text to the left border of the cell. It is not possible to center the text in the cells.
The meaning of that grid was to be used as a map/legend, for the bits on the left side.
Would you elaborate a bit on the practical use of that what you described?

**Reply by gigli69** [reply]
It's a visual enhancement, because it not practical to deal with the binary values on the left side.
